pkg/ipam/address/address.go

```diff
@@ -74,6 +74,18 @@ type ReserveRandom struct {
 	LocationID string `json:"location_identifier"`
 	VlanID     string `json:"vlan_identifier"`
 	Count      int    `json:"count"`
+
+	// PrefixID limits the potential addresses to a specific prefix
+	// within the specified VLAN. Defaults to all prefixes within the VLAN.
+	PrefixID string `json:"prefix_identifier,omitempty"`
+
+	// IPVersion limits the potential addresses to a specific IP version.
+	// Defaults to v4 & v6.
+	IPVersion int `json:"ip_version,omitempty"`
+
+	// ReservationPeriod specifies how many seconds the addresses should be reserved.
```

Comment:
Two questions come to my mind:

1. What happens if we pass a negative value?
2. What happens after the reservation period? If I assign the IP to a VM, is it going to be removed after the period again?

I think a sentence like "If the IP address hasn't been assigned to a resource within the period, it is released again." could help here.

Reply:
1. Changed to `uint`
2. Will add the proposed comment (changed to plural "addresses")